modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/saml/AbstractSAMLSSOTestCase.java

```diff
@@ -280,6 +284,12 @@ public void createUser(SAMLConfig config) {
         }
     }
 
+    public void createApplicationForSAMLArtResolve(SAMLConfig config, String appName) throws Exception {
```

Comment:
missing Javadoc

Reply:
Fixed https://github.com/wso2/product-is/pull/9477/commits/70d73b570425b27e4ff199a8c93aec0c7fb47645